frontend/src/lib/components/Forms/Score.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+	import { formFieldProxy, type SuperForm } from 'sveltekit-superforms/client';
+	import { ProgressRadial } from '@skeletonlabs/skeleton';
+	import { RangeSlider } from '@skeletonlabs/skeleton';
+	import * as m from '$paraglide/messages';
+	import type { AnyZodObject } from 'zod';
+	import { displayScoreColor } from '$lib/utils/helpers';
+
+	export let label: string | undefined = undefined;
+	export let field: string;
+
+	export let min_score: number;
+	export let max_score: number;
+
+	export let form: SuperForm<AnyZodObject>;
+	const { value, errors, constraints } = formFieldProxy(form, field);
+	
+	$: scoringEnabled = $value === null ? false : true
+
+	function preventNull(value: number){
+		if(value === null){
+			return 0
+		}
+		return value * 100 / max_score
+	
+	}
```

Comment:
The name of this function implies it is meant to prevent null values while it computes a percentage. This is misleading